### ISSUE TITLE
Merging GPU autoscheduler

### ISSUE THREAD

**Issue by BachiLi** [halide/Halide#5602]
We should at some point port the GPU autoscheduler (in standalone_autoscheduler_gpu) to the main branch.
@aekul

**Comment by neoskypig** [comment]
What is the plan of this issue?

**Comment by aekul** [comment]
Hopefully it will be merged in the near future but there are no concrete plans at the moment.

**Comment by ryanstout** [comment]
@aekul Is there concern of bitrot on the `standalone_autoscheduler_gpu` branch? I would offer to help if I was a bit more familiar with halide internals. I would hate to see that branch not get merged in.

**Comment by steven-johnson** [comment]
Is this still active?

**Comment by ryanstout** [comment]
@steven-johnson I would love to see it land at some point, not holding my breath at this point though :-)

**Comment by steven-johnson** [comment]
> @steven-johnson I would love to see it land at some point, not holding my breath at this point though :-)

What are the obstacles to moving forward with it, as far as you know?

**Comment by ryanstout** [comment]
@steven-johnson I think it's just really stale at this point. I wasn't involved in it though, so thats an outsider's look at it. I'm sure there's a lot of details to get it brought up before it could be merged. Thanks!

**Comment by TH3CHARLie** [comment]
yeah the biggest obstacle is that Luke or myself could not find time to push this forward before, I will meet with Andrew and Derek later today and I will bring this up to them and see if we can have a swift plan to merge this.

**Comment by aekul** [comment]
I haven't had a lot of time to work on this but have been preparing a basic PR lately that should be ready sometime this week. It doesn't attempt to merge/de-duplicate the code that's shared with `adams2019` so if that is a blocker, then more time/work will be required.

**Comment by ashishUthama** [comment]
Been meaning to try this out too but was unable to hack out something from the branch to use.
I would welcome a quick "v.1" version as an end user.

**Comment by antonysigma** [comment]
Hi I noticed the PR #6856 has been merged around 4 days ago. What is the release schedule? `v16.0` ? 

Asking because I am compiling the Halide binary from source, which I found is not scalable in a long run.

**Comment by abadams** [comment]
Yes, this should go out in Halide 16

**Comment by abadams** [comment]
> [!NOTE]
> **Automated triage:** this issue was flagged by Claude as likely safe to close, for the reason below. The reasoning was checked against the current source, and a second pass tried and failed to refute it, but no human verified this specific text. **If the reason is wrong, please reopen the issue and say so** — some fraction of these are expected to be wrong.

**Reason (CLOSE_FIXED):** The ask was to port the GPU autoscheduler from the `standalone_autoscheduler_gpu` branch onto main. It landed as `src/autoschedulers/anderson2021` via PR #6856 and shipped in Halide 16; it has since been maintained (test fixes, PTX gating).

**Evidence:** `src/autoschedulers/anderson2021/` exists in-tree; follow-up commits `831fd1a47` ("Fix RDom usage in anderson2021_test_apps_autoscheduler"), `f11e80da0`, `b45064704` ("Skip tests for anderson2021 if PTX is not enabled"), and `bc30d6f83`.

Closing as done: the GPU autoscheduler was merged as `src/autoschedulers/anderson2021` (PR #6856) and shipped in Halide 16.
